### PR TITLE
Adding dvntylr to _pages

### PR DESCRIPTION
**Pull request by dvntylr**

### CODE REVIEW

File: _pages/dvntylr.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: dvntylr
+pageUrl: dvntylr.github.io
+timestamp: 2017-04-09
+---
```
Comment (ridsjin) [inline _pages/dvntylr.yaml]: I see a strange character at the end of this line. I believe that is causing the issue!

**Comment by sjdeak** [comment]
don't be discouraged

**Comment by beardofedu** [comment]
@dvntylr I modified your file to add an additional line to the end of your file.

**Comment by beardofedu** [comment]
@dvntylr I identified the issue, to pass the test you need to delete the `_pages/maxa31.yaml` file in your branch. The hyperlink associated with maxa31 doesn't work because they deleted the repository. After you delete the file and add the deletion of the file as a commit and push it up, your tests should pass.

**Comment by crichID** [comment]
@dvntylr it looks like the tests are failing because your web page no longer exists or pages has not been enabled on the repository. If you have created your repository, please go to the repository **settings** and scroll down to the **pages** section. Using the drop-down, select the **master** branch to render your website. When you have finished this, your web page will display at the link shown at the top of the pages section. 

I am going to go ahead and close this pull request since your page is not currently enabled. If you would like to finish this activity, please feel free to re-open this pull request after you have enabled pages on your repository.

**Pull request closed without merging**